packages/lab/stories/static-list/static-list.qa.stories.tsx

```diff
@@ -0,0 +1,97 @@
+import type { Meta, StoryFn } from "@storybook/react";
+import { QAContainer, type QAContainerProps } from "docs/components";
+
+import {
+  Button,
+  FlexLayout,
+  StackLayout,
+  StatusIndicator,
+  Text,
+} from "@salt-ds/core";
+import { DeleteIcon } from "@salt-ds/icons";
+import { StaticList, StaticListItem } from "@salt-ds/lab";
+
+export default {
+  title: "Lab/Static List/QA",
```

Comment:
Should end in Static List QA

Reply:
Hey Josh, sorry I did not understand your comment.

Reply:
```suggestion
  title: "Lab/Static List/Static List QA",
```